graphql/image_test.go

```diff
@@ -21,6 +21,30 @@ func init() {
 	testutil.Setup()
 }
 
+func TestOperatingSystem(t *testing.T) {
+	config := New("/graphql")
+	ctx := getContext(t)
+	testConfig := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, testConfig, "TestOperatingSystem")
+	require.NoError(t, testConfig.RuntimeEnvironments.Set(ctx))
+	ami := "ami-0f6b89500372d4a06"
+	image := model.APIImage{
+		AMI: &ami,
+	}
+	opts := thirdparty.OSInfoFilterOptions{
+		AMI:  ami,
+		Name: "Kernel",
+	}
+	res, err := config.Resolvers.Image().OperatingSystem(ctx, &image, opts)
+	require.NoError(t, err)
+	require.NotNil(t, res)
+	require.Len(t, res.Data, 1)
+	require.NotNil(t, res.Data[0])
+	assert.Equal(t, utility.FromStringPtr(res.Data[0].Name), "Kernel")
```

Comment:
nit: the expected value is the first parameter to `assert` functions, the actual value is the second one. It's not a big deal, but the tests will show the wrong expected/actual values if the assertion fails, which can be a little confusing.

Reply:
erm ive been assuming the opposite for so long... I will fix all of these in the corresponding files 🙏